jazkarta/shop/at.py

```diff
@@ -22,10 +26,9 @@ class ExtendedBooleanField(ExtensionField, atapi.BooleanField):
 class ExtendedFloatField(ExtensionField, atapi.FloatField):
     pass
 
-
+@adapter(IATProduct)
+@implementer(ISchemaExtender)
```

Comment:
Isn't this just going to fail if the `ImportError` above is triggered? Should we make some mock interfaces on ImportError?